tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -97,22 +99,39 @@ def generate_dependencies(list, package_name, version):
         # UAP10.0.16299, This is the earliest release of the OS that supports .NET Standard apps
         list.append('<group targetFramework="UAP10.0.16299">')
         list.append('</group>')
+        # Support Native C++
+        list.append('<group targetFramework="native">')
```

Comment:
should we add this for uap also?

Reply:
@jstoecker @smk2007 This change maybe causes #6249